### PR TITLE
Add Styled components in SkillUsageCard, SkillRating, SkillHistory, SkillFeedbackCard

### LINKED ISSUE #2269: Add Styled components in SkillUsageCard, SkillRating, SkillHistory
**Actual Behaviour**

Not Present

**Expected Behaviour**

Add Styled components in SkillUsageCard, SkillRating, SkillHistory

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2269, #2270

Changes: 
- [x] Fix CountryWiseSkillUsageCard List #2270
- [x] Add Styled components in SkillUsageCard, SkillRating, SkillHistory, SkillFeedbackCard

Demo Link: https://pr-2271-fossasia-susi-web-chat.surge.sh

**Screenshot**

Screenshot 2019-06-14 at 3 44 55 AM

### CODE REVIEW

**Comment by fragm3** [comment]
@akshatnitd How to manage the margin styles(talking about `<Paper className="margin-b-md margin-t-md">`) used in Paper, should we create a styled component for it and reuse it everywhere?

**Comment by akshatnitd** [comment]
@fragm3 Yes, go ahead and make a shared styled component for it. Try to keep it as generic as possible.

**Comment by fragm3** [comment]
@akshatnitd done, please review. Will take up SkillPage and SkillFeedbackPage later

File: src/components/shared/Typography.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+
+export const Title = styled.h1`
+  padding-bottom: 1rem;
+  margin: ${props => (props.marginTop ? '20px' : '0px')};
+`;
+
+export const SubTitle = styled.p`
+  font-weight: normal;
+  font-size: ${props => (props.p ? '1rem' : '1.5rem')}
```
Comment (akshatnitd) [inline src/components/shared/Typography.js]: Why have you named the prop as `p`?

**Comment by fragm3** [comment]
@akshatnitd it looked cleaner, p === 1 rem font size. If you want me to change it, I can rename it to size as well

**Comment by fragm3** [comment]
@akshatnitd I have used size instead of p

**Comment by fragm3** [comment]
@akshatnitd I tried the list in different resolutions it shows up fine. I have attached a sc as well. Can you describe which browser and resolution you have?

**Pull request merged**